### PR TITLE
=pro #18744 Disable generation of javadoc for akka-bench-jmh

### PR DESCRIPTION
**Pull request by johanandren**
Refs #18744

### CODE REVIEW

File: project/AkkaBuild.scala [hunk 1]
```diff
@@ -119,6 +119,8 @@ object AkkaBuild extends Build {
       persistence, distributedData,
       testkit
     ).map(_ % "compile;compile->test;provided->provided")
+  ).settings(
+    sources in doc in Compile := List()
```
Comment (ktoso) [inline project/AkkaBuild.scala]: please remove here and in stead in `akka-bench-jmh/build.sbt` add this line: `AkkaBuild.dontPublishSettings`

**Comment by ktoso** [comment]
Valid fix, but not the style we're doing nowadays :)
LGTM after my comment applied

File: project/AkkaBuild.scala (hunk 1 above)
Comment (johanandren) [inline project/AkkaBuild.scala]: That's already set (and does not work). Hmm.

Comment (ktoso) [inline project/AkkaBuild.scala]: Oh, that might be of interest to @2m

Comment (ktoso) [inline project/AkkaBuild.scala]: Duh I'm so blind, sorry.

Comment (ktoso) [inline project/AkkaBuild.scala]: How about we add to AkkaBuild this:

```scala
  val dontPublishDocsSettings = Seq(
    sources in doc in Compile := List()
  )
```

and that we include in build.snt

Comment (johanandren) [inline project/AkkaBuild.scala]: :+1:

**Comment by drewhk** [comment]
unrelated failure, LGTM

**Comment by patriknw** [comment]
LGTM

**Pull request merged**